mkdocs.yml

```diff
@@ -2,8 +2,14 @@ docs_dir: docs/book
 site_dir: docs/html
 pages:
     - index.md
```

Comment:
Can you add "Home" (`- Home: index.md`) here? I think, then the entry in the subnavigation is skipped.